msteams-platform/concepts/deploy-and-publish/appsource/prepare/Test-preview-for-monetized-apps.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Test preview for monetized apps 
+author: v-ypalikila
+description: Create and test SaaS Preview offers for Teams app before pushing the offer live.
+ms.topic: conceptual
+ms.author: v-ypalikila
+ms.localizationpriority: medium
+keywords: teams apps SaaS offer preview offer  app source preview
+---
+
+# Test preview for monetized apps
+
+As a developer, you can create a Software as a Service (SaaS) offer and test the end-to-end purchase experience of an offer in Teams before you publish the offer. Users who are added as the preview audience for the Teams app can test the preview offer, see [ Add a preview audience for a SaaS offer](/azure/marketplace/create-new-saas-offer-preview).
+
+## Configure your app with the Preview offer ID
+
+After adding the preview audience to your SaaS offer, link the offer ID to your Teams app for users to see your subscription plans in the Teams store.
+
+1. Go to [Developer Portal](https://dev.teams.microsoft.com/).
```

Comment:
Ensure log in with a "preview audience" account in Developer Portal to be able to see the preview offer plans. Otherwise, non-preview audience user can't see the preview offer plans.